### DISCUSSION TITLE
Ώρα έναρξης επόμενης ανεπίσημης ΟΣΣ (Σάββατο 5/11)

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#23] [Q&A] [answered]
Καλησπέρα σας, κύριε Χωριανόπουλε.

Θα μπορούσαμε να ξεκινήσουμε την επόμενη ΟΣΣ, που έχουμε κανονίσει για το Σάββατο 5/11, λίγο αργότερα από τις 16:00 που ισχύει συνήθως; Έχω επιβεβαιώσει ότι δεν υπάρχει αντίρρηση από τους υπόλοιπους συναδέλφους για να ξεκινήσουμε στις 17:00 (ή 17:30 αν σας εξυπηρετεί καλύτερα).

Σας ευχαριστώ για την κατανόηση,
Αγνή Φωσκόλου

**Comment by epidrome** [comment, marked as answer]
ούτε εγώ μπορώ στις 4μμ, αλλά θα προσπαθήσω για τις 6μμ, θα βγει ανακοίνωση, 

καθώς έχουμε κάνει ήδη 6ώρες συνάντησης, καλό θα ήταν να χρησιμοποιήσουμε και την ασύγχρονη επικοινωνία λίγο περισσότερο, όπως βλέπω να γίνεται τώρα

**Comment by afoscolou** [comment]
Καλησπέρα σας.

Τελικά θα έχουμε συνάντηση αύριο στις 18:00;

**Reply by epidrome** [reply]
ναι #31
